### ISSUE TITLE
I can't log to onlyoffice workspace. Login screen is looping.

### ISSUE THREAD

**Issue by mastipgithub** [ONLYOFFICE/CommunityServer#398]
I updated my docker instance od debian by workspace-install script from 11 version to 12.
I used command bash workspace-install.sh -u true -ics true -ids true -icp true -ims true

I can't log to onlyoffice workspace. Login screen is looping. 
I enter loging details and screen is looping and i can write loging details again. Nothing changes.
![weblog](https://user-images.githubusercontent.com/4679142/174281387-3aef0292-71cf-4ce0-bad7-825bf7d05a79.PNG)

**Comment by Carazyda** [comment]
Hello @mastipgithub Please describe in more details how did you install Onlyoffice Workspace? Have you changed your database connection settings? Are you using an external database? try running the update sql script on the database onlyoffice for the new version - /var/www/onlyoffice/Sql/onlyoffice.upgradev120.sql.

**Comment by mastipgithub** [comment]
First version of portal I installed by this method https://helpcenter.onlyoffice.com/installation/workspace-install-docker-integrated.aspx. This was time when there was no control panel and elastic search containers. I made folders for binding volumes, installed mysql 5.7 db docker, document server, community server. I have naver used email server but it was installed.
Then I upgraded portal by workspace-script.

**Comment by mastipgithub** [comment]
How to start this script?
I have logged to community server container by command docker exec -it containername bash
Found script file in folder /var/www/onlyoffice/Sql/

**Comment by Carazyda** [comment]
Try this command (password and user you can see in the /var/www/onlyoffice/WebStudio/web.connections)
`mysql -u root -pmy-secret-pw -h onlyoffice-mysql-server onlyoffice</var/www/onlyoffice/Sql/onlyoffice.upgradev120.sql`

**Comment by mastipgithub** [comment]
It works. Thank You Man. You made my day.

**Comment by Carazyda** [comment]
Sorry, I didn't see that you had version 11.5 You also need to run this script:
`mysql -u root -pmy-secret-pw -h onlyoffice-mysql-server onlyoffice</var/www/onlyoffice/Sql/onlyoffice.upgradev116.sql`

**Comment by mastipgithub** [comment]
It works now. So every after workspace-script update I need start new script from this to update database?

**Comment by Carazyda** [comment]
No, this is done automatically when the community server is updated. Why it didn't work for you, I'm at a loss to answer. Perhaps you still changed the default settings for connecting to the database. And we don't change the database that often, only in major updates.
